src/components/ec-user-info/ec-user-info.spec.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-use-before-define */
+import { mount } from '@vue/test-utils';
+import EcUserInfo from '@/components/ec-user-info/ec-user-info.vue';
+import { withMockedConsole } from '../../../tests/utils/console';
+
+const user = {
+  name: 'Ebury Demo 2',
+  profileUrl: '/profile',
+  gravatar: 'https://www.gravatar.com/avatar/e07fd6efc70ccc63bbc3a3e27b81b29e?d=mm&s=200',
+};
+
+describe('EcUserInfo', () => {
+  it('should render as expected', () => {
+    const wrapper = mount(EcUserInfo, {
+      propsData: {
+        user,
+      },
+    });
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should emit a toggle event', () => {
+    const wrapper = mount(EcUserInfo, {
+      propsData: {
+        user,
+      },
+    });
+    wrapper.vm.toggle();
```

Comment:
We should not be triggering actions using `.vm`. What if the toggle function is not attached in the template? Instead of that, do this:
```
wrapper.find('.icon').trigger('click');
```